Formalize MusicBrainz reference, explain why no tags are defined (#237)
ietf-wg-cellar/flac-specification: flac.md

### Standard field names

> Only one standard field name is defined: the channel mask field, in (#channel-mask). No other field names are defined because the applicability of any field name is strongly tied to the content it is associated with. For example, field names useful for describing files that contain a single work of music would be unusable when labeling archived broadcasts, recordings of any kind, or a collection of music works. Even when describing a single work of music, different conventions exist depending on the kind of music: orchestral music differs from music by solo artists or bands.

Reviewer: ```suggestion
Only one standard field name is defined: the channel mask field, in (#channel-mask). Other field names are free-form field names with their meaning being outside the scope of this specification because the applicability of any field name is strongly tied to the content it is associated with. For example, field names useful for describing files that contain a single work of music would be unusable when labeling archived broadcasts, recordings of any kind, or a collection of music works. Even when describing a single work of music, different conventions exist depending on the kind of music: orchestral music differs from music by solo artists or bands.
```

Or

```suggestion
Only one standard field name is defined: the channel mask field, in (#channel-mask). Other field names are free-form field names with their meaning being outside the scope of this specification.
```

(I am not sure that there is a need of more details here, it is just outside of this spec)

Author: In last call review and IESG reviews the term 'hand-waiving' was used a few times, I believe at least once in this context. I think there is a need to explain why no tags are formalized: not because we are too "lazy" to define this but because there is a real reason why defining a one-size-fits-all approach for field names is impossible: use cases are too different. It is not out-of-scope per se, if it was straightforward I would have included it.

> Despite the fact that no field names are formally defined, there is a general trend among devices and software capable of FLAC playback that are meant to play music. Most of those recognize at least the following field names:

Reviewer: ```suggestion
Despite the fact that no field names are formally defined, there is a general trend among devices and software capable of FLAC playback that are meant to play music. An implementation MAY recognize the following field names:
```

For defining the level of RFC 2119 recommendation (here: optional)

Author: As part of the shepherds review, a lot of RFC 2119 keywords were removed because they were in informative parts of the specification (like the section conceptual overview), and they don't really belong there. I think the same goes here: this sentence gives a hint on what a specific set of implementations often do, which isn't normative at all. So I think RFC 2119 keywords aren't really useful here?

- Title: name of the current work.
- Artist: name of the artist generally responsible for the current work. For orchestral works, this is usually the composer; otherwise, it is often the performer.
- Album: name of the collection the current work belongs to.

For a more comprehensive list of possible field names suited for describing a single work of music in various genres, the list of tags used in the MusicBrainz project, see [@MusicBrainz], is suggested.

> <reference anchor="MusicBrainz" target="http://picard-docs.musicbrainz.org/en/variables/variables.html">

Reviewer 2: `http://` > `https://`

Author: ```suggestion
<reference anchor="MusicBrainz" target="https://picard-docs.musicbrainz.org/en/variables/variables.html">
```
